test/terraform/gcp/main.tf

```diff
@@ -0,0 +1,97 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 6.0"
+    }
+  }
+}
+
+provider "google" {
+  project = var.project_id
+  region  = var.region
+}
+
+module "materialize" {
+  source = "github.com/MaterializeInc/terraform-google-materialize?ref=v0.1.1"
+
+  project_id = var.project_id
+  region     = var.region
+  prefix     = "tf-gcp-test"
+
+  database_config = {
+    tier     = "db-custom-2-4096"
+    version  = "POSTGRES_15"
+    password = var.database_password
+  }
+
+  labels = {
+    environment = "simple"
+    example     = "true"
+  }
+
+  install_materialize_operator = false
+
+  # Once the operator is installed, you can define your Materialize instances here.
+  materialize_instances = var.materialize_instances
```

Comment:
nit: as we are installing the Materialize CRDs with `kubectl` directly, we don't need those in here.